stream-chat-android-client/src/androidTest/java/io/getstream/chat/android/client/utils/TestInitCallback.kt

```diff
@@ -34,9 +35,9 @@ internal class TestInitCallback : Call.Callback<ConnectionData> {
 
     override fun onResult(result: Result<ConnectionData>) {
         if (result.isSuccess) {
-            data = result.data()
+            data = result.getOrThrow()
         } else {
-            error = result.error()
+            error = result.streamErrorOrNull()
```

Comment:
```suggestion
            error = result.streamErrorOrThrow()
```
I assume this is supposed to be something like `result.streamErrorOrThrow()` like `error()` before. WDYT?

```kotlin
    public fun error(): ChatError {
        return checkNotNull(error) {
            "Result is successful, not an error. Check result.isSuccess before reading the error."
        }
    }
```